cloud_saver:
  ros__parameters:
    cloud_topic: "/point_cloud"
    export_path: "/tmp/cloud.npy"
